### PR TITLE
Added environment variables

### PR DESCRIPTION
**Pull request by jtj9817**
Added config_vars.env for the production-stage front-end component of NodeChat. PR fix for #52

### CODE REVIEW

File: config_vars.env [hunk 1]
```diff
@@ -0,0 +1 @@
+LOCAL_HOST_FRONTEND = 'http://localhost:9876'
```
Comment (joshghent) [inline config_vars.env]: Let's rename this to be "PORT" and call this file "sample.env"

Comment (joshghent) [reply config_vars.env]: Create another variable called "URL"

File: src/loginPage/loginPage.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ class LoginPage extends Component {
     constructor(props) {
         super(props);
 
-        this.socket = io('http://localhost:9876');
+        this.socket = io(process.env.LOCAL_HOST_FRONTEND);
```
Comment (joshghent) [inline src/loginPage/loginPage.js]: Please default the url and port to localhost:9876 or throw an error if we do not have this config item passed

**Review by joshghent — CHANGES_REQUESTED** [review]

**Review by dnguneratne — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -17,6 +17,7 @@
     "url": "git@github.com:joshghent/NodeChat.git"
   },
   "dependencies": {
+    "dotenv": "^6.1.0",
```
[the remark this answers is not in the document]
Comment (jtj9817) [reply package.json]: Just quickly skimmed that doc. So based on understanding, I just have to rename the variables with a prefix of 'REACT_APP' and I can remove the dotenv package? Once I get confirmation on this, I'll make the necessary changes accordingly.

**Review by joshghent — APPROVED** [review]

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (dnguneratne) [reply package.json]: Yes, that's right. You'll also need to rename `config_vars.env` file to `.env`.

The environment variable can be accessed like this: `process.env.REACT_APP_[variable_name]`.

[the remark this answers is not in the document]
Comment (joshghent) [reply package.json]: Nah .env should be gitignored. .env has the personal tokens and private stuff. That is why you need to provide a sample.env that is then duplicated and renamed to `.env`

**Review by joshghent — CHANGES_REQUESTED** [review]
Could you please fix up the environment variable names please? And add the .env file to the gitignore

**Pull request closed without merging**